### PR TITLE
Cc baker wintime

### PR DESCRIPTION
**Pull request by EmilLa1**
## Purpose
Introduce the `win-time` baker query.
## Changes

_Describe the changes that were needed.

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [ ] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: source/mainnet/net/references/query-node.rst [hunk 1]
```diff
@@ -318,3 +319,21 @@ following command:
...
+Earliest time a baker may be expected to bake a block
+-----------------------------------------------------
+
+.. code-block:: console
+
+    $concordium-client baker win-time 1
+    Baker 1 is expected to bake no sooner than:
+    Thu, 26 Oct 2023 07:01:26 UTC  (in 34s 699ms)
+
+Show the earliest time the baker may be expected to bake a block.
```
Comment (abizjak) [inline source/mainnet/net/references/query-node.rst:335]: Can you put some more words to this. There are quite a lot of qualifiers needed here about the meaning of the response. We have some of this in the API documentation.

File: source/mainnet/net/references/query-node.rst [hunk 2]
```diff
@@ -318,3 +319,21 @@ following command:
...
+
+-  ``BAKER-ID``: Baker ID to query
+-  ``local-time``: Display time in the local time zone (instead of UTC).
+-  ``poll``: Repeatedly poll for the latest time..
```
Comment (abizjak) [inline source/mainnet/net/references/query-node.rst:339]: I don't think reproducing the help text is needed here. But I would mention explicitly the `--poll` option in text above.

**Comment by abizjak** [comment]
Are you going to address this @EmilLa1 ?

**Comment by EmilLa1** [comment]
Should any of this be renamed to "validator" ? @dg-concordium

**Comment by abizjak** [comment]
The new client will have `validator win-time`.

File: source/mainnet/net/references/query-node.rst [hunk 3]
```diff
@@ -318,3 +319,31 @@ following command:
...
+
+Baker queries
+==============
+
+Earliest time a baker may be expected to bake a block
+-----------------------------------------------------
+
+.. code-block:: console
+
+    $concordium-client baker win-time 1
```
Comment (abizjak) [inline source/mainnet/net/references/query-node.rst:331]: Please use validator terminology.

Comment (EmilLa1) [reply source/mainnet/net/references/query-node.rst]: Updated the text.

File: source/mainnet/net/references/query-node.rst [hunk 4]
```diff
@@ -323,27 +323,27 @@ also pass a block hash to the command to query a specific block.
 Validator queries
 =================
 
-Earliest time a validator may be expected to make a block
+Earliest time a validator may be expected to produce a block
 -----------------------------------------------------
```
Comment (abizjak) [inline source/mainnet/net/references/query-node.rst:327]:
```suggestion
------------------------------------------------------------
```

**Review by abizjak — APPROVED** [review]
To be merged with the release of client 6.2.

**Review by dg-concordium — APPROVED** [review]

**Pull request merged**